SYCL/lit.cfg.py

```diff
@@ -387,11 +387,11 @@
 config.substitutions.append( ('%ACC_CHECK_PLACEHOLDER',  acc_check_substitute) )
 
 if config.sycl_be == 'ext_oneapi_cuda' or (config.sycl_be == 'ext_oneapi_hip' and config.hip_platform == 'NVIDIA'):
-    config.substitutions.append( ('%sycl_triple',  "nvptx64-nvidia-cuda" ) )
+    config.substitutions.append( ('%sycl_triple',  "nvptx64-nvidia-cuda-syclmlir" ) )
```

Comment:
Isn't this going to fail with the current development status (same for AMD)?

I assume we currently only support lowering from SYCL-MLIR to SPIR-V, not GCN or PTX. 
If that is the case, we should maybe mark the tests as `UNSUPPORTED` on those platforms. 

Alternatively, we could introduce a feature `cgeist` or `sycl-mlir` (or any other name), that we `REQUIRES` and which is currently only enabled on SPIR-V based targets. With this approach, we could avoid to touch every single test-case once we enable the other targets in SYCL-MLIR.

Reply:
Our current SYCL-MLIR-based compilation flow support lowering from C -> Clang AST -> SYCL dialect -> LLVM dialect -> LLVMIR ... I don't have an AMD or NVIDIA device to check, but shouldn't it logically work as well?